### ISSUE TITLE
Update plain button input group example

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3134]
The plain button example should mimic the existing popover example instead of the search example. The input `type` and `aria-label` on both the input and button should be updated.
